### DISCUSSION TITLE
UX of differences in saving changes (manual/automatic) feels strange (aka: Blue dot + 'Save changes')

### DISCUSSION THREAD

**Discussion by 0xdevalias** [community/community#4712] [Projects and Issues]
When making a change to some of the settings on a new issues beta project, it seems that they may be only made 'locally' (or for my account?) initially, and show a blue dot like the following to apparently indicate 'unsaved changes':

Tab view:
![image](https://user-images.githubusercontent.com/753891/126085965-6ffb55d7-da5a-4b96-a931-761ec00eefb2.png)

Tab 'settings' view:
![image](https://user-images.githubusercontent.com/753891/126085969-b982cf97-5a5c-4608-a74f-e2e12ee7d6de.png)

It took me a little bit to figure out what this blue dot was intending to show/mean, and that I had to explicitly 'save' those changes. It might be nice if this was made clearer in some way, perhaps even as a 'first time use tutorial' with 'callout bubbles or similar?

I also found it a little 'jarring' that in some places, changes I make seem to be instantly saved/published (eg. project settings -> Project name), whereas some others need to be manually saved (eg. project settings -> Fields (eg. status) -> Save options), adding a new column on the 'board' view/renaming an existing one seems to save instantly, and then there are the 'blue dot' changes needing to be saved manually as well.

**Comment by velocity23** [comment]
I know this is an old thread but as someone who just got access to the beta recently I 100% agree. I couldn't figure out (especially in dark mode) what the little blue dot wanted me to do for a good few minutes, which is terrible UX imo.

**Reply by bduffany** [reply]
I also found this blue dot to be very easy to miss (I reported this here: https://github.com/github/feedback/discussions/6490) -- with this type of UX pattern, there is usually a dialog that says "You have unsaved changes, are you sure you want to exit" before leaving the page -- I think that would help a bit. But TBH, even with that dialog, I wouldn't catch that the blue dot means "unsaved changes." If such a dialog were to popup, I would probably think, huh, what unsaved changes do I have? Shouldn't everything be saved already?

I think it would be better to just auto-save instead of this unsaved-changes pattern. Other spreadsheet-style webapps save everything as you go, so I think people have the ingrained expectation that everything will be auto-saved.

**Reply by Logistics92** [reply]
True that
